ChangeLog

```diff
@@ -207,10 +207,14 @@ Release date: TBA
 
   Refs pylint-dev/pylint#8598
 
+* Fix false positives for ``no-member`` and ``invalid-name`` when using the ``_name_``, ``_value_`` and ``_ignore_`` sunders in Enums.
+
+  Closes pylint-dev/pylint#9015
+
 
 What's New in astroid 2.15.7?
 =============================
-Release date: TBA
+Release date: 2023-09-23
```

Comment:
I thought this should be brought by the maintenance branch merge back in main ? (https://github.com/pylint-dev/astroid/blob/main/doc/release.md#releasing-a-patch-version)